Docs/src/ogre-shadows.md

```diff
@@ -397,15 +400,386 @@ Non-PCF block. Note that after doing this, the uSTexWidth and
+
+The default projection used when rendering shadow textures is a uniform frustum. 
+This is pretty straight forward but doesn't make the best use of the space in the shadow map 
+since texels closer to the camera will be larger, resulting in 'jaggies'. 
+There are several ways to distribute the texels in the shadow texture differently.
+Ogre is provided with several alternative shadow camera setups:
+ - Ogre::FocusedShadowCameraSetup: Implements the uniform shadow mapping algorithm in focused mode. 
+ - LiSPSM (Ogre::LiSPSMShadowCameraSetup): Implements the Light Space Perspective Shadow Mapping Algorithm.
+ - PSSM (Ogre::PSSMShadowCameraSetup): Parallel Split Shadow Map (PSSM) shadow camera setup.
+
+ - Plane Optimal (Ogre::PlaneOptimalShadowCameraSetup): Implements the plane optimal shadow camera algorithm.
+
+These Shadow Camera Setups can be enabled for the whole Scene with SceneManager::setShadowCameraSetup 
+or per light with Light::setCustomShadowCameraSetup
+
+The following shows how to activate Plane Optimal Shadow Mapping given
+some pointer to a MovablePlane and a pointer to a light.
+```cpp
+Ogre::MovablePlane *movablePlane = new Ogre::MovablePlane( Ogre::Vector3::UNIT_Y, 0 );
+Ogre::Entity *movablePlaneEntity = mSceneMgr->createEntity( "movablePlane", "Floor.mesh" );
+Ogre::SceneNode *movablePlaneNode = mSceneMgr->getRootSceneNode()->createChildSceneNode("MovablePlaneNode");
 movablePlaneNode->attachObject(movablePlaneEntity);
-light->setCustomShadowCameraSetup(Ogre::ShadowCameraSetupPtr(planeOptShadowCamera));
-```
\ No newline at end of file
+Ogre::ShadowCameraSetupPtr shadowCameraSetup = Ogre::PlaneOptimalShadowCameraSetup::create(movablePlane);
+light->setCustomShadowCameraSetup(Ogre::ShadowCameraSetupPtr(shadowCameraSetup));
+```
+
+Another example, using LiSPSM Camera Setup:
+```cpp
+Ogre::ShadowCameraSetupPtr shadowCameraSetup = Ogre::LiSPSMShadowCameraSetup::create();
+mSceneMgr->setShadowCameraSetup(shadowCameraSetup);
+```
+
+For big scenes with directional lights one of the better performing Shadow Camera Setups is PSSM.
+A PSSM shadow system uses multiple shadow maps per light and maps each texture into a region of space, 
+progressing away from the camera. As such it is most appropriate for directional light setups.
+A more in depth explanation can be found in the wiki: [Parallel Split Shadow Mapping](https://wiki.ogre3d.org/Parallel+Split+Shadow+Mapping)
+
+```cpp
+// General scene setup
+mSceneMgr->setShadowTechnique(Ogre::SHADOWTYPE_TEXTURE_ADDITIVE_INTEGRATED);
+mSceneMgr->setShadowTextureCasterMaterial(Ogre::MaterialManager::getSingletonPtr()->getByName("PSSMCaster"));
+mSceneMgr->setShadowTextureSelfShadow(true);
+mSceneMgr->setShadowFarDistance(3000);
+// 3 textures per directional light (PSSM)
+mSceneMgr->setShadowTextureCountPerLightType(Ogre::Light::LT_DIRECTIONAL, 3);
+mSceneMgr->setShadowTextureCount(3);
+mSceneMgr->setShadowTextureConfig(0, 2048, 2048, Ogre::PF_FLOAT16_RGB);
```

Comment:
this wastes 66% of shadow texture memory. Use PF_DEPTH16 instead. That way you can also use the default Shadow caster material.